src/schoology-extractor/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
```

Comment:
I'm unclear why this doesn't include `"python.pythonPath": ".venv\\Scripts\\python.exe"`. Are you running a single poetry env for all projects?

Reply:
I did it by mistake, I'll replace line two with the pythonPath